src/js/components/Settings/AdminMisc.vue

```diff
@@ -25,19 +25,32 @@
 		<CheckboxRadioSwitch :checked.sync="hideLogin" type="switch">
 			{{ t('polls', 'Hide login option in public polls') }}
 		</CheckboxRadioSwitch>
+		<CheckboxRadioSwitch :checked.sync="autoArchive" type="switch">
+			{{ t('polls', 'Archive closed polls automatically') }}
+		</CheckboxRadioSwitch>
+		<div v-if="autoArchive" class="settings_details">
+			<span>{{ t('polls', 'Days after closing, the polls shall be archived:') }}</span>
```

Comment:
```suggestion
			<span>{{ t('polls', 'After how many days are the closed polls to be archived:') }}</span>
```

Reply:
Argh. Wrong button. This wording sounds like a question.